vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -1784,12 +1784,17 @@ void AdsImpl::SustainAdNotificationInteractionIfNeeded() {
     BLOG(INFO) << "Failed to sustain ad notification interaction, domain for "
         "the focused tab does not match the last shown ad notification for "
             << last_shown_ad_notification_.target_url;
+
+    last_sustained_ad_notification_url_ = "";
```

Comment:
could this be moved at the top, so that we don't need to do it twice? As far as I can see all paths set this

Reply:
ah yes, was put here originally as I was resetting `last_shown_ad_notification_` which was needed here. Changing now

Reply:
Fixed